dv/uvm/common_project_cfg.hjson

```diff
@@ -5,6 +5,7 @@
   project:          ibex
 
   // These keys are expected by dvsim.py, so we have to set them to something.
+  book:              bogus.book.domain
```

Comment:
I tried to revendor locally and it doesn't give me this line

Reply:
Oh! Now I remember! I had to do this manually. The problem is that the latest version of dvsim (which we're pulling from OpenTitan) now depends on that key in a file that it imports (and smushes into Python datastructures in a rather horrible way).

The dependency comes from [OpenTitan commit ff2314b16b919aff2d1149947ed6e2432f2a27b7](https://github.com/lowRISC/opentitan/commit/ff2314b16b919aff2d1149947ed6e2432f2a27b7) from back in July.

Can you see a cleaner way to do this update?

Reply:
I think the solution is fine, but you'd want to mention this in the commit message.

Reply:
Silly me: there's a much easier way! I've just split this into two commits. The first adds this line to `common_project_cfg.hjson` and the second is the vendoring commit. Hopefully this will make everything a bit more obvious.